app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -222,36 +222,47 @@ class TableVisualization extends Visualization {
             this.agGridOptions.api.setPinnedTopRowData([extraRow])
         }
         this.agGridOptions.api.setRowData(rowData)
-        this.agGridOptions.api.sizeColumnsToFit()
+        this.updateTableSize(this.dom.getAttributeNS(null, 'width'))
     }
 
-    updateTableSize() {
-        if (this.tabElem !== undefined) {
-            const width = this.dom.getAttributeNS(null, 'width')
-            const height = this.dom.getAttributeNS(null, 'height')
-            const tblViewStyle = `width: ${width}px; height: ${height}px; overflow: scroll;`
-            this.tabElem.setAttributeNS(null, 'style', tblViewStyle)
-            /**
-             * By observation, calling `sizeColumnsToFit()` outside of the `requestAnimationFrame`
-             * callback function sometimes has no effect. This is because the browser's reflow
-             * mechanism may prevent `sizeColumnsToFit()` from obtaining the latest width of
-             * `this.tabElem` internally, resulting in the computed column width being the same
-             * as before and having no effect. The callback function of `requestAnimationFrame`
-             * is called when the page is ready for repainting, so all DOM elements have already
-             * been updated at that point. Therefore, the `sizeColumnsToFit` function in the
-             * `requestAnimationFrame` callback can ensure that it gets the latest width of
-             * `this.tabElem`.
-             */
-            window.requestAnimationFrame(() => {
-                this.agGridOptions.api.sizeColumnsToFit()
-            })
+    updateTableSize(clientWidth) {
+        if (!this.agGridOptions) {
```

Comment:
The same comment as to the previous review - please add here an explanation what it does and why we are returning here.